influxql/parser_test.go

```diff
@@ -1627,24 +1627,14 @@ func TestParser_ParseStatement(t *testing.T) {
 		{
 			s: `CREATE DATABASE testdb`,
 			stmt: &influxql.CreateDatabaseStatement{
-				Name:                  "testdb",
-				IfNotExists:           false,
-				RetentionPolicyCreate: false,
-			},
-		},
-		{
-			s: `CREATE DATABASE IF NOT EXISTS testdb`,
-			stmt: &influxql.CreateDatabaseStatement{
-				Name:                  "testdb",
-				IfNotExists:           true,
+				Name: "testdb",
```

Comment:
It actually is the format.  I've never noticed it before, but apparently if the variables are really long it won't line them up.  I tested locally and it keeps fmt'ing them back to this format.